bukkit/src/main/java/net/skinsrestorer/bukkit/skinrefresher/SpigotSkinRefresher.java

```diff
@@ -63,17 +63,23 @@ public SpigotSkinRefresher(SkinsRestorer plugin, SRLogger log) throws Initialize
             try {
                 removePlayerEnum = ReflectionUtil.getEnum(playOutPlayerInfo, "EnumPlayerInfoAction", "REMOVE_PLAYER");
                 addPlayerEnum = ReflectionUtil.getEnum(playOutPlayerInfo, "EnumPlayerInfoAction", "ADD_PLAYER");
-            } catch (Exception e) {
+            } catch (Exception e1) {
                 try {
-                    // Forge
-                    removePlayerEnum = ReflectionUtil.getEnum(playOutPlayerInfo, "Action", "REMOVE_PLAYER");
-                    addPlayerEnum = ReflectionUtil.getEnum(playOutPlayerInfo, "Action", "ADD_PLAYER");
-                } catch (Exception e1) {
-                    Class<?> enumPlayerInfoAction = ReflectionUtil.getNMSClass("EnumPlayerInfoAction", null);
-
-                    // 1.8 or below
-                    removePlayerEnum = ReflectionUtil.getEnum(enumPlayerInfoAction, "REMOVE_PLAYER");
-                    addPlayerEnum = ReflectionUtil.getEnum(enumPlayerInfoAction, "ADD_PLAYER");
+                    Class<?> enumPlayerInfoActionClass = Class.forName("net.minecraft.network.protocol.game.PacketPlayOutPlayerInfo$EnumPlayerInfoAction");
```

Comment:
can you add more comments? i am sure what this is for is this only for cardboard?